src/lib/y2network/interface_config_builders/dummy.rb

```diff
@@ -0,0 +1,18 @@
+require "yast"
+require "y2network/interface_config_builder"
+
+module Y2Network
+  module InterfaceConfigBuilders
+    class Dummy < InterfaceConfigBuilder
+      def initialize
+        super(type: "dummy")
+      end
+
+      def save
```

Comment:
In general miss documentation in `InterfaceConfigBuilder`methods, in particular do not expected to return sometimes the set of an attribute and sometimes `nil`.. specially when more than one attribute is responsibility of a specialized class.